array_api_tests/test_sorting_functions.py

```diff
@@ -5,7 +5,7 @@
 from hypothesis import strategies as st
 from hypothesis.control import assume
 
-from xptests.typing import Scalar, ScalarType, Shape
+from .typing import Scalar, ScalarType, Shape
```

Comment:
An aside: we need to avoid absolute imports like this, so that the module can easily be vendored. Maybe we should add a simple meta-test for this.

Reply:
Yep this was just an unintentional mistake